.github/workflows/build_wheels_aarch64_linux.yml

```diff
@@ -0,0 +1,60 @@
+name: Build Aarch64 Linux Wheels
+
+on:
+    pull_request:
+    push:
+      branches:
+        - nightly
+        - main
+        # Release candidate branch look like: v1.11.0-release
+        - v[0-9]+.[0-9]+.[0-9]+-release+
+      tags:
+        # Release candidate tag look like: v1.11.0-rc1
+        - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+        - v[0-9]+.[0-9]+.[0-9]+
+    workflow_dispatch:
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
+    cancel-in-progress: true
+
+jobs:
+  generate-matrix:
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    with:
+      package-type: wheel
+      os: linux-aarch64
+      test-infra-repository: pytorch/test-infra
+      test-infra-ref: main
+      with-cuda: disable
+  build:
+    needs: generate-matrix
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - repository: pytorch/FBGEMM
+            smoke-test-script: ""
+            pre-script: ../.github/scripts/nova_prescript.bash
+            post-script: ../.github/scripts/nova_postscript.bash\
```

Comment:
nit: remove trailing `\`. also don't need the `../` since it starts looking in the project root directory

Reply:
I force `cd fbgemm_gpu` in `env-var-script` if it's in project root directory. Will the script gets read before or after the `env-var-script` is executed?